src/Common/Command/RegisterApiTestClientCommand.php

```diff
@@ -0,0 +1,128 @@
+<?php
+
+/**
+ * CreateReleaseChangelogCommand Is a helper utility to create a human readable changelog for a release
+ * based on the milestone name.  It relies on the naming convention of prefixing each issue with the category
+ * feat: for feature, bug: for bug, refactor: for changes, and chore: for chores.  If no category is specified
+ * it will default to bug.  It will also look for the label developers and if it is present it will categorize the
+ * issue as a developer issue.  That way regular users can see feature changes separated out from developer specific issues
+ * such as api, code refactoring, etc.
```

Comment:
another minor thing, seems like some of the command code was copied from the changelog utility but thinking these docblocks weren't updated for their purpose here